kerchunk/codecs.py

```diff
@@ -66,6 +66,19 @@ def decode(self, buf, out=None):
 numcodecs.register_codec(FillStringsCodec, "fill_hdf_strings")
 
 
+class FletcherDummyFilter(numcodecs.abc.Codec):
+    codec_id = "fletcher_null"
+
+    def decode(self, buff, out=None):
+        return buff[:-4]
+
+    def encode(self, buf):
+        pass
```

Comment:
Wouldn't you want to append 4 empty bytes here, just in case someone tried to use it in a round-trip context?

Reply:
Hm, I'm not sure. Maybe raising NotImplemented is the correct thing to do, as the output can't be used by anything that really does the check.